### PR TITLE
Add K8s manifests to documentation

### PR DESCRIPTION
**Pull request by Aesop7**

### CODE REVIEW

**Review by shamoon — APPROVED** [review]
Thanks, just some small notes if you wouldn't mind

File: src/pages/en/installation/k8s.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
+---
+title: Kubernetes Installation
+description: Install on Kubernetes
+layout: ../../../layouts/MainLayout.astro
+---
+
+## Method 1:  Helm
```
[the remark this answers is not in the document]
Comment (Aesop7) [reply src/pages/en/installation/k8s.md]: Yep, I like that better too.  fixed with latest commit

**Comment by Aesop7** [comment]
Fixed a link I noticed in the Vercel preview as well

**Comment by shamoon** [comment]
Cheers thanks

**Pull request merged**